src/apps/users/services/core.py

```diff
@@ -93,7 +95,7 @@ async def send_password_recovery(
             subject="Password reset",
```

Comment:
Can you update `Password reset` to use localized strings? I don't think we have this for the backend so it could instead be a ticket to fix this in soon.